<ui version="4.0" >
 <class>RaidClass</class>
 <widget class="QWidget" name="RaidClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Raid</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="maximumSize" >
        <size>
         <width>420</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Book Antiqua</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="title" >
        <string>RAID Status</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>RAID Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Hdd Capacity:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>3TB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>DiskCount:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15" >
            <item>
             <widget class="QLabel" name="label_21" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Total:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_20" >
              <property name="text" >
               <string>0.00TB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16" >
            <item>
             <widget class="QLabel" name="label_23" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Avalible:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_22" >
              <property name="text" >
               <string>0.00TB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17" >
            <item>
             <widget class="QLabel" name="label_25" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Used:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_24" >
              <property name="text" >
               <string>0.00TB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" >
            <item>
             <widget class="QLabel" name="label_9" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>RAID Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" >
            <item>
             <widget class="QLabel" name="label_11" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Disk 1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" >
            <item>
             <widget class="QLabel" name="label_13" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Disk 2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" >
            <item>
             <widget class="QLabel" name="label_14" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Disk 3:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_15" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" >
            <item>
             <widget class="QLabel" name="label_17" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Disk 4:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11" >
            <item>
             <widget class="QLabel" name="label_19" >
              <property name="minimumSize" >
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Disk 5:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>groupBox_2</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="maximumSize" >
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Book Antiqua</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="title" >
        <string>RAID Tools</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13" >
          <item>
           <spacer name="horizontalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton" >
            <property name="maximumSize" >
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>Rebuild RAID5</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser" >
          <property name="maximumSize" >
           <size>
            <width>300</width>
            <height>800</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_26" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="font" >
      <font>
       <family>Book Antiqua</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title" >
      <string>Content Travel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" >
      <item>
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="title" >
         <string>RAID Array</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6" >
         <item>
          <widget class="QTreeWidget" name="treeWidget_HDD_info" >
           <property name="font" >
            <font>
             <family>Book Antiqua</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <column>
            <property name="text" >
             <string>Status</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>FilmName</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>RecvTime</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>FilmLength</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>RecvSegment</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>TotalSegment</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14" >
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4" >
          <property name="icon" >
           <iconset resource="cinecastui.qrc" >
            <normaloff>:/CineCastUi/Resources/Reflash.png</normaloff>:/CineCastUi/Resources/Reflash.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3" >
          <property name="icon" >
           <iconset resource="cinecastui.qrc" >
            <normaloff>:/CineCastUi/Resources/up.png</normaloff>:/CineCastUi/Resources/up.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2" >
          <property name="icon" >
           <iconset resource="cinecastui.qrc" >
            <normaloff>:/CineCastUi/Resources/down.png</normaloff>:/CineCastUi/Resources/down.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5" >
          <property name="icon" >
           <iconset resource="cinecastui.qrc" >
            <normaloff>:/CineCastUi/Resources/delete.png</normaloff>:/CineCastUi/Resources/delete.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5" >
        <property name="title" >
         <string>Removeable HDD</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5" >
         <item>
          <widget class="QTreeWidget" name="treeWidget_HDD_info_2" >
           <property name="font" >
            <font>
             <family>Book Antiqua</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <column>
            <property name="text" >
             <string>Status</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>FilmName</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>RecvTime</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>FilmLength</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>RecvSegment</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>TotalSegment</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
        <zorder>treeWidget_HDD_info_2</zorder>
        <zorder></zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="cinecastui.qrc" />
 </resources>
 <connections/>
</ui>
